packages/core/src/model/chain/velas.ts

```diff
@@ -0,0 +1,29 @@
+import { Chain } from '../../constants'
+
+export const Velas: Chain = {
+  chainId: 106,
+  chainName: 'Velas Mainnet',
+  isTestChain: false,
+  isLocalChain: false,
+  multicallAddress: '0x55c77eEba2b891c7f940cE4C3d9Fcd6915c12082',
+  multicall2Address: '0x324f25e6eEB13D45DF559B7326d631e34Fd5ceDF',
+  getExplorerAddressLink: (address: string) => `https://evmexplorer.velas.com/address/${address}`,
+  getExplorerTransactionLink: (transactionHash: string) => `https://evmexplorer.velas.com/tx/${transactionHash}`,
+}
+
+export const VelasTestnet: Chain = {
+  chainId: 111,
+  chainName: 'Velas Testnet',
```

Comment:
Can You verify chainId of this testnet?
I check and https://chainlist.org/ showed me that 111 is Ether Lite Chain.

Reply:
Of course, check and in the Velas Network Testnet documentation it has the ChainId 111
https://docs.velas.com/clusters/#testnet

Reply:
Okay, perfect.
Run pnpm lint:fix, because this is a problem with linter and we can merge:)

Reply:
I'm going to close the pull request, because I created another following another branch, because apparently I made a mess